### ISSUE TITLE
Enable cross-repository epics

### ISSUE THREAD

**Issue by krystof-k** [cloudaper/epics-action#2]

**Comment by bs** [comment]
This is my first time looking at a GH action ...

Would you need to break the scope of `repo: github.context.repo.repo` and match across a repo/# key?

Thanks for your work <3

**Comment by krystof-k** [comment]
Yes, something like that. You would also need to set up the workflow in all repositories (or in those which will contain the partial issues).
